packages/shared/apis/PortableExperiences.ts

```diff
@@ -34,10 +40,10 @@ export class PortableExperiences extends ExposableAPI {
   @exposeMethod
   async kill(pid: PortableExperienceUrn): Promise<boolean> {
     const parcelIdentity: ParcelIdentity = this.options.getAPIInstance(ParcelIdentity)
-    const portableExperience: PortableExperienceHandle | undefined = await getPortableExperience(pid)
+    const portableExperience = getRunningPortableExperience(pid)
 
     if (!!portableExperience && portableExperience.parentCid === parcelIdentity.cid) {
-      return await killPortableExperienceScene(pid)
+      store.dispatch(removeDebugPortableExperience(pid))
```

Comment:
kill will always return false since we remove the `return` key from here. Is that expected ?